Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/Internal/NamespaceSynchronizationConfig.swift

```diff
@@ -13,7 +13,10 @@ import StitchCoreSDK
  */
 final class NamespaceSynchronization: Sequence, Codable {
     static func filter(namespace: MongoNamespace) -> Document {
-        return ["namespace": try? BSONEncoder().encode(namespace)]
+        guard let namespaceDoc = try? BSONEncoder().encode(namespace) else {
+            return ["namespace": BSONNull()]
+        }
+        return ["namespace": namespaceDoc]
```

Comment:
[style nit] could we get rid of the guard block and simplify this to `try? BSONEncoder().encode(namespace) ?? BSONNull()`?